CMakeLists.txt

```diff
@@ -123,7 +126,7 @@ else()
 endif()
 
 # install MIVisionX docs -- {ROCM_PATH}/share/doc/mivisionx/
-install(DIRECTORY docs DESTINATION share/doc/mivisionx)
+install(DIRECTORY docs DESTINATION ${CMAKE_INSTALL_DATADIR}/doc/mivisionx)
```

Comment:
We need to replace this with DOCDIR instead of DATADIR

Reply:
as discussed since the project name is difference from the folder name, not added this right now.